### DISCUSSION TITLE
Run BARK in the cloud

### DISCUSSION THREAD

**Discussion by igorcb123** [serp-ai/bark-with-voice-clone#15] [Q&A]
I tried to run the voice clone in google collab and apart from being very slow 🐢 in the end it didn't work because although I was able to clone the voice I could never use it

That's why I cloned the project locally, finally the result was very very slow (probably because I don't have GPU), I haven't tried the local voice cloning part yet

It has occurred to me that perhaps it can be run in the Google Cloud or Azure Cloud or some other but I would like to know the opinion of the creators

I think I could make a web service for my own consumption that receives the parameters by post and returns the wav file as a result, would this be faster?

I also have no idea what cloud could work best and I also don't want something that is very expensive for me

What I finally want to do is clone my own voice to generate tts in Spanish to create free educational content

What do you suggest?

**Comment by devinschumacher** [comment]
@igorcb123 We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik
